src/config/dynamic-theme-fixes.config

```diff
@@ -2748,6 +2748,28 @@ div.output_png img
 
 ================================
 
+kaytrip.com
+
+INVERT
+.date
+.jd_ss
+.liuchengt
+.nApp
+.nTel
+.n_app
+.tt_t
+.wmap
+
+CSS
+#imageField {
+    background-color: transparent !important;
+}
+.jd_xl {
+    filter: none !important;
```

Comment:
4 spaces please.

Reply:
@Gusted I believe it's 4 spaces already ._.

Reply:
I can't count woops.